tests/lib/rules/no-unused-prop-types.js

```diff
@@ -784,54 +764,6 @@ ruleTester.run('no-unused-prop-types', rule, {
       ].join('\n')
     }, {
       code: [
-        'const statelessComponent = (props) => {',
```

Comment:
Got it. Makes sense. Is ti ok if I delete these tests though. Do you agree that this rule technically doesn't support this use case.